src/style.css

```diff
@@ -1 +1,88 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.nav {
+  display: flex;
+  height: 60px;
+  justify-content: flex-end;
+  text-transform: uppercase;
+}
+
+.header {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 50px;
+}
+
+.link {
+  color: black;
+  text-decoration: none;
+  margin-left: 20px;
```

Comment:
Reset margin left for first link
![image](https://github.com/mate-academy/layout_moyo-header/assets/54936044/7d2eeb5e-a0b7-4487-9731-69b923584cdd)